packages/xo-server-usage-report/report.html.tpl

```diff
@@ -222,7 +226,7 @@
           </td>
         <tr>
         <tr>
-          <td>Net transmission:</td>
+          <td>Network TX:</td>
```

Comment:
Are *RX* and *TX* clear enough?
Wouldn't *input* and *output* be better?

Reply:
No RX and TX are fine.